applied-crypto-examples/src/tutorials.rs

```diff
@@ -132,3 +133,38 @@ pub fn merlin_basics_tutorial() {
     println!("Alternatively, by defining the same domain labels and byte encodings for objects we're concerned about");
     println!("we can define a consistent hashing scheme for all objects we find interesting.");
 }
+
+pub fn merlin_non_interactive_proof_tutorial() {
```

Comment:
This should be interactive, and give the user more information. However, this can be done in a followup PR.

Reply:
This is true, the next followup PR will addresses this